addons/io_hubs_addon/components/utils.py

```diff
@@ -12,7 +18,9 @@ def add_component(obj, component_name):
 
     component_class = get_component_by_name(component_name)
     if component_class:
-        update_gizmos()
+        if 'create_gizmo' in component_class.__dict__:
```

Comment:
I guess there is some case that I'm missing but as all components inherit from `HubsComponents` won't this always be true?